helios-client/src/main/java/com/spotify/helios/common/JobValidator.java

```diff
@@ -150,7 +150,11 @@ public JobValidator(final boolean shouldValidateJobHash,
         errors.add("Volume path is not absolute: " + path);
         continue;
       }
-      if (!isNullOrEmpty(source) && !source.startsWith("/")) {
+      if (isNullOrEmpty(source)) {
```

Comment:
it seems like before, a null or empty source was allowed. Now this will cause an error. Am I correct on that and is that change intentional?